docs/tutorials/customize_runtime.md

```diff
@@ -0,0 +1,321 @@
+# Tutorial 5: Customize Runtime Settings
+
+## Customize Optimization Settings
+
+### Customize optimizer supported by PyTorch
+
+We already support to use all the optimizers implemented by PyTorch, and the only modification is to change the `optimizer` field of config files.
+For example, if you want to use `ADAM` (note that the performance could drop a lot), the modification could be as the following.
+
+```python
+optimizer = dict(type='Adam', lr=0.0003, weight_decay=0.0001)
+```
+
+To modify the learning rate of the model, the users only need to modify the `lr` in the config of optimizer.
+The users can directly set arguments following the [API doc](https://pytorch.org/docs/stable/optim.html?highlight=optim#module-torch.optim) of PyTorch.
```

Comment:
this line is pretty vague. could u provide an example?

Reply:
An Example is given as the following

Reply:
cool